src/platform/qt/scripting/ScriptingView.h

```diff
@@ -22,6 +22,7 @@ Q_OBJECT
 private slots:
 	void submitRepl();
 	void load();
+	void recent();
```

Comment:
I'd recommend the name `loadRecent` instead, as `recent` isn't a verb and lacks clarity of the action in question